component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -2389,4 +2408,37 @@ private void setUserClaimValues(AuthenticatedUser authenticatedUser, Map<String,
                     String.format("Failed to update user claims for user : %s.", authenticatedUser), e);
         }
     }
+
+    private long getUnlockTimeInMilliSeconds(AuthenticatedUser authenticatedUser) throws AuthenticationFailedException {
```

Comment:
Shall we add a method comment with the behavior of the method? Why we return `0` likewise? WDYT?

Reply:
fixed in b87fff7